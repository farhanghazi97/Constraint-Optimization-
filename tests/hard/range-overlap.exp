m1:wed 11am
cost:0
