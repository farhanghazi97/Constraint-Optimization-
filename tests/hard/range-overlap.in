meeting, m1
domain, m1, tue 12pm-wed 12pm, hard
domain, m1, wed 11am-thu 10am, hard
