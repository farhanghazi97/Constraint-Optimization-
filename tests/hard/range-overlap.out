m1:wed 12pm
cost:0
